src/generators/genlua.ml

```diff
@@ -477,6 +525,10 @@ and gen_expr ?(local=true) ctx e = begin
 		spr ctx "]";
 	| TBinop (op,e1,e2) ->
 		gen_tbinop ctx op e1 e2;
+	| TField ({eexpr = TCall _} as e, f) when is_multireturn ctx e ->
+		print ctx "_G.select(%i, " (multi_value_idx_lookup_expr ctx e (field_name f));
+		gen_value ctx e;
+		spr ctx ")";
```

Comment:
If we're immediately accessing a multireturn field value that is the result of a method call, then just use lua's select() method.  This avoids having to create any temporary variables.